straxen/plugins/event_processing.py

```diff
@@ -653,8 +665,10 @@ def setup(self):
 
         self.s2_map = InterpolatingMap(
             get_cmt_resource(self.run_id,
-                             tuple([*self.config['s2_xy_correction_map']]),
-                             fmt='text'))
+                             tuple(['suffix',
```

Comment:
Why do we need this `suffix` this looks somewhat odd to me?

Reply:
No not for the moment, but in my opinion CMT must be reworked. It has become one of the most complicated parts in straxen.

Reply:
We need the suffix because now s2 maps are position dependent, and in order to use only one single option for three different maps i.e. s2_map_mlp, s2_map_gcn, s2_map_cnn we add that suffix, then at the end this works with one single straxen option `self.config['s2_xy_correction_map']`. Is either that or we modified straxen to set every configuration option for every position algorithm